tests/integration/conftest.py

```diff
@@ -450,3 +449,41 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers", "smoke: mark test as part of smoke test suite"
     )
+
+
+@pytest.fixture
+def create_linode():
```

Comment:
Can we use a different fixture name?
Best practice is to use descriptive name or use a prefix noun instead of verb.